tools/labs/Makefile

```diff
@@ -32,8 +32,12 @@ copy: $(YOCTO_IMAGE)
 	rmdir $(TEMPDIR)
 
 docker-docs:
-	cd docs && docker-compose build
-	cd docs && docker-compose run docs-build bash -c "cd /linux/tools/labs && make docs"
+	cd docker && docker-compose build docs-build
+	cd docker && UID=$(shell id -u) GID=$(shell id -g) docker-compose run docs-build bash -c "cd /linux/tools/labs && make docs"
+
+docker-kernel:
+	cd docker && docker-compose up -d kernel-build
+	cd docker && docker-compose exec kernel-build bash
```

Comment:
why not docker-compose run instead of up + exec?

Reply:
Because docker-compose run would drop people in different containers for every `make docker-kernel`. I want people to be able to drop in the same container in order to connect to the qemu VM using minicom.